packages/core/src/app/shipping/SingleShippingForm.tsx

```diff
@@ -111,10 +112,50 @@ class SingleShippingForm extends PureComponent<
     ) {
         super(props);
 
-        const { updateAddress } = this.props;
-
         this.debouncedUpdateAddress = debounce(
             async (address: Address, includeShippingOptions: boolean) => {
+                const { 
```

Comment:
My suggestion would be here to create another debounce function based on the experiment and use the experiment as a switch in `updateAddressWithFormData` function.